tests/console/yast2_tftp.pm

```diff
@@ -35,8 +35,7 @@ sub run {
     assert_screen 'yast2_tftp-server_configuration_enabled';
 
     # provide a new TFTP root directory path
-    send_key 'alt-t';    # select input field
-    assert_screen 'yast2_tftp-server_configuration_chdir';
+    send_key_until_needlematch 'yast2_tftp-server_configuration_chdir', 'alt-t', 2, 3;    # select input field
```

Comment:
Could you please put the comment that we do it mainly because somehow first key press is lost and we improve stability here by retrying.